### PR TITLE
TravisCI: Add new packages in Travis whitelist

### PR DESCRIPTION
**Pull request by jayvdb**
Adds:
- flawfinder
- libperl-critic-perl

### CODE REVIEW

**Comment by jayvdb** [comment]
This has negligible effect on build times, however it moves us closer to sudo-less Travis - when we get there, the build booting delays will be much shorter.

[Current](https://travis-ci.org/coala/coala-bears/jobs/186262910#L1271): 0 upgraded, 82 newly installed, 0 to remove and 67 not upgraded.

[This PR](https://travis-ci.org/coala/coala-bears/jobs/186395938#L1532): 0 upgraded, 34 newly installed, 0 to remove and 67 not upgraded.

**Comment by Mixih** [comment]
Can't we squash the two package addition commits into one?

File: .travis.yml [hunk 1]
```diff
@@ -71,6 +72,9 @@ before_install:
   # Remove Ruby directive from Gemfile as this image has 2.2.5
   - sed -i '/^ruby/d' Gemfile
   - mkdir -p ~/.apt-cache
+  - sudo add-apt-repository -y ppa:marutter/rdev
+  - sudo add-apt-repository -y ppa:staticfloat/juliareleases
+  - sudo add-apt-repository -y ppa:ondrej/golang
```
Comment (sims1253) [inline .travis.yml]: It sounded like you moved these three from somewhere else. Shouldn't this include the removal from the other place?

Comment (jayvdb) [inline .travis.yml]: They are still in deps.sh for CircleCI.  Need to mention that in commit message?

Comment (sims1253) [inline .travis.yml]: well this was just me trying to review the pr despite having very little experience with what you did :) 
Soooo I can't really answer your question. Maybe put it there but if having them in deps.sh is expected/wanted then I see no need to point it out.

Comment (sils) [inline .travis.yml]: maybe rather make a switch to deps.sh instead of duplicating the code?

Comment (Mixih) [inline .travis.yml]: well using whitelisted packages should speed up the build, and I don't think they've been whitelisted in circle yet...

Comment (jayvdb) [inline .travis.yml]: Circle doesnt have a whitelist.

But these three lines are duplication, and are not whitelisted, and whitelist requests are usually very slow to be processed.

I can add a bool env var `USE_PPA_TRAVIS_TRUSTY`, so these three lines appear only once in `.ci/deps.sh`, or `.ci/deps.sh` could even use `USE_PPA` and `TRAVIS` combined to activate only these three.

I could also convert `USE_PPA` to be a string like `USE_PPA="marutter/rdev staticfloat/juliareleases ondrej/golang` in `.travis.yml`.

Comment (sils) [inline .travis.yml]: the last proposal sounds best in terms of maintainability IMO

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1197/commits/9a44cc16cbe319e2a18b6f63fd60f0f4f1bdfc99

**Comment by sils** [comment]
https://github.com/coala/coala-bears/pull/1197/commits/6fbb2a7964f4c00d3028acd915715c154872500c needs work

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1197/commits/4342b68427dea6028be092c6e62304e04bb75dd1

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1197/commits/eee68ef7b872b541046c39d1e09d80abd8a19b82 https://github.com/coala/coala-bears/pull/1197/commits/eee68ef7b872b541046c39d1e09d80abd8a19b82

**Comment by sils** [comment]
@jayvdb feel free to rebase, reack and merge

**Comment by jayvdb** [comment]
reack 7f19055 f002b89

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**